### ISSUE TITLE
Add additional release channels

### ISSUE THREAD

**Issue by awentzel** [microsoft/fast#4626]
Enhance our nightly publishing strategy to include additional channels as FAST packages mature over time using [beachball](https://microsoft.github.io/beachball/).

#### Publish to NPM
Links for inspiration.
- https://github.com/danillouz/til-npm-release-channels

#### Dependency
[This](https://github.com/microsoft/beachball/issues/429) feature is required for FAST to add additional tags with metadata.  Metadata capability is not yet available but described in that issue.  Once this is complete (perhaps we contribute) we can resume channel implementation.

#### Release channels / tags
Include additional changes to our existing Nightly and Instant publishing process.  Naming is still a work in progress.

**Note that `dev`, `beta`, and `latest` will be used.

* **On Demand** automated from master to `latest` ~~or `release`~~ based on major package bumps.
* **Nightly** automated from master to `dev` ~~or `canary`~~
* **Weekly** automated from master to `beta` ~~or `next`~~
* * Tags with both `dev` and `beta`
* * ~~Tags with both `canary` and `next`~~
* * Format aligns with Semver 2. For example [major.minor.patch-suffice+yyyymmdd] 
* **Monthly** and/or as desired manually to `stable`

For example:
```
yarn publish --tag `dev` or `1.1.13-dev+20210407`
```

#### Source Code Branches
We may also want to consider using an actual `/releases` branch, such that anytime a release is merged into this branch, the CI/CD process fires to deploy to Azure.  https://docs.microsoft.com/en-us/azure/static-web-apps/overview  We do have to understand a little more detail about static web apps to ensure this will work.  For example, we also want to consider what happens in the context of containers, local development and portability, along with what if we end up having non-static web apps, then what?  

- https://swa-playground.wassim.dev/p
- https://github.com/Azure/static-web-apps-cli
- https://docs.microsoft.com/en-us/azure/static-web-apps/
- https://docs.microsoft.com/en-us/azure/static-web-apps/overview

Resources:
For details https://docs.npmjs.com/adding-dist-tags-to-packages#:~:text=Publishing%20a%20package%20with%20a%20dist-tag%20%C2%A7%20By,navigate%20to%20the%20root%20directory%20of%20your%20package 

This PR is related to #3889 #3543 #3660

**Comment by Ibrahimmaga** [comment]
we need to set up a meeting on tagging again. Last time it was a little confusing whether to use canary, dev or latest

**Comment by janechu** [comment]
Couple clarifying questions:
- What's the need with `beta` if it's also automated like our `dev` release? Do we plan to have some additional checks in place before publishing to `beta` thereby making it more stable?
- Not sure we should have stable releases as automated releases, what are others thoughts?

Can we get a list of blockers? I see the `status:blocked` tag, also there was mention of some missing meta-data, if there is an issue filed can we get a link to that for tracking purposes?

**Comment by janechu** [comment]
Closing this due to lack of activity.
